beacon-chain/state/state-altair/deprecated_getters.go

```diff
@@ -0,0 +1,13 @@
+package state_altair
+
+import pbp2p "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+
+// PreviousEpochAttestations is not supported for HF1 beacon state.
+func (b *BeaconState) PreviousEpochAttestations() []*pbp2p.PendingAttestation {
+	panic("PreviousEpochAttestations is not supported for hard fork 1 beacon state")
```

Comment:
Panic is bad. Can we change the API to return `([]*pbp2p.PendingAttestation, error)`?